riseagain47/EPIC-Mail
# Ch refactor server folder contents #164577037

#### What does this PR do?
- Refactor server folder contents


#### Description of Task to be completed?
- Refactor server folders and files in order to attain a better and cleaner file structure.
Have the following:
- Controllers folder
- Models folder
- Separated server.js into two different files server.js and app.js
- Confirm that the TDD setup is still working

#### How should this be manually tested?
- User can view the server folder to observe the changes implemented.
- User can run `npm test` on the terminal to observe TDD implementation
- User should run `npm start` on the terminal to observe the server running.

#### Any background context you want to provide?
This PR adds aesthetic value to the file structure.

#### What are the relevant pivotal tracker stories?
- Chore #164577037 - refactor server folder contents
